app/main.py

```diff
@@ -2,114 +2,324 @@
 from sanic import response
 from kafka import KafkaProducer
 from kafka.errors import KafkaError
-from pymongo import MongoClient
-import json
 import logging
-
+from app.models import *
+from math import ceil
+import json
 
 api = Blueprint('api')
 DEFAULT_TOPIC = "audio-recordings"
+ITEMS_PER_PAGE = 20
 
-client = MongoClient('mongodb://localhost:27017/')
 
-my_db = client.linguadb
+responseError = {"message": "Invalid Payload."}
+responseListObjects = {"page": 1, "results": [],
+                       "total_results": 0, "total_pages": 0}
 
 
 @api.route("/")
 def index(request):
     return response.json({"message": "welcome to Lingua API"})
 
 
+@api.route("/languages", methods=["POST"])
+def saveLanguages(request):
+    if request.method == "POST":
+        postdata = request.json
+        requiredFields = ["language", "code", "type"]
+        if  not set(requiredFields) >= set( postdata):
+            return response.json(responseError, status=400)
+        try:
+            new_lang = Languages(
```

Comment:
Nit: There is a mix a of camelCase and snake_case in your code let's settle on one.